src/main/java/constant/WinLottoInfo.java

```diff
@@ -0,0 +1,48 @@
+package constant;
+
+import java.util.Arrays;
+import java.util.List;
+import model.LottoNumbers;
+import model.WinLotto;
+
+public enum WinLottoInfo {
+    FIRST(6, false, 2000000000),
+    SECOND(5, true, 30000000),
+    THIRD(5, false, 1500000),
+    FOURTH(4, false, 50000),
+    FIFTH(3, false, 5000),
```

Comment:
```suggestion
    FIRST(6, false, 2_000_000_000),
    SECOND(5, true, 30_000_000),
    THIRD(5, false, 1_500_000),
    FOURTH(4, false, 50_000),
    FIFTH(3, false, 5_000),
```
숫자는 가독성을 위해 _로 구분지을 수 있습니다.